### ISSUE TITLE
Enhancement request: searchable comments field

### ISSUE THREAD

**Issue by TobyHFerguson** [redis-developer/redis-microservices-demo#47]
I think it would be good to have an application where users could enter text comments, and those comments could be searched. We might consider doing this as an enrichment of the data from the db, rather than by modifying the db itself, thus further demonstrating how Redis can be wrapped around a legacy system - this would also allow us to talk about durability!

**Comment by tgrall** [comment]
Yes extending the model should be done, and comments is what I used in other demo. But did not have the time to do it yet

* [x] Create Comment for movies (insert, delete, list)
* [ ] Search

**Comment by tgrall** [comment]
@TobyHFerguson 
![Screenshot 2020-11-06 at 15 23 51](https://user-images.githubusercontent.com/541250/98376938-7e136f80-2044-11eb-8547-5f1c7f437a6a.png)
